ovs/lib/alba.py

```diff
@@ -610,6 +605,80 @@ def add_cluster(alba_backend_guid, abm_cluster=None, nsm_clusters=None):
         AlbaController.checkup_maintenance_agents.delay()
         alba_backend.invalidate_dynamics('live_status')
 
+    @staticmethod
+    def can_set_auto_cleanup():
+        # type: () -> bool
+        """
+        Return if it is possible to set the auto-cleanup
+        :return: True if possible else False
+        :rtype: bool
+        """
+        storagerouter = StorageRouterList.get_storagerouters()[0]
+        return StorageRouter.ALBA_FEATURES.AUTO_CLEANUP in storagerouter.features['alba']['features']
+
+    @staticmethod
+    def set_auto_cleanup(alba_backend_guid, days=30, config=None):
+        # type: (str, int) -> None
+        """
+        Set the auto cleanup policy for an ALBA Backend
+        :param alba_backend_guid: Guid of the ALBA Backend to set the auto cleanup for
+        :type alba_backend_guid: str
+        :param days: Number of days to wait before cleaning up. Setting to 0 means disabling the auto cleanup
+        and always clean up a namespace after removing it
+        :type days: int
+        :param config: Arakoon configuration of the backend (optional)
+        :type config: str
+        :return: None
+        :rtype: NoneType
+        """
+        if not isinstance(days, int) or 0 > days:
+            raise ValueError('Number of days must be an integer > 0')
+        if config is None:
+            alba_backend = AlbaBackend(alba_backend_guid)
+            config = Configuration.get_configuration_path(key=alba_backend.abm_cluster.config_location)
+        if AlbaController.can_set_auto_cleanup():
+            # Check if the maintenance config was not set (disabled)
+            maintenance_config = AlbaCLI.run(command='get-maintenance-config', config=config)
```

Comment:
Any specific reason why you use the alba cli to fetch the config? below, you have function for this purpose